doc: ActionCable overview grammar and wording fixes [ci-skip] (#44065)
rails/rails: guides/source/action_cable_overview.md

### Consumers

> The client of a WebSocket connection is called the *consumer*. In Action Cable

Reviewer: We can add a comma here:

```suggestion
The client of a WebSocket connection is called the *consumer*. In Action Cable,
```

### Passing Parameters to Channels

> When creating a subscription, you can pass parameters from the client-side to the
> server-side. For example:

Reviewer: Let's keep this as-is:

```suggestion
You can pass parameters from the client-side to the server-side when creating a
subscription. For example:
```

The appearance example was all about exposing server functionality to
client-side invocation over the WebSocket connection. But the great thing

> about WebSockets is that it's a two-way street. So now, let's show an example

Reviewer: I think this reads fine without the comma, but, if we do add that comma, we should add another:

```suggestion
about WebSockets is that it's a two-way street. So, now, let's show an example
```

## Deployment

> A combination of WebSockets and threads powers Action Cable. The

Reviewer: Let's keep this as-is:

```suggestion
Action Cable is powered by a combination of WebSockets and threads. Both the
```

framework plumbing and user-specified channel work are handled internally by

> utilizing Ruby's native thread support. This means you can use all your typical

Reviewer: ```suggestion
utilizing Ruby's native thread support. This means you can use all your existing
```

Rails models with no problem, as long as you haven't committed any thread-safety sins.

The Action Cable server implements the Rack socket hijacking API,
thereby allowing the use of a multithreaded pattern for managing connections

> internally, irrespective of whether the application server is multithreaded or not.

Author: I saw it on line 890 and I thought it was the right way 😅
